core/src/net/sf/openrocket/util/TestRockets.java

```diff
@@ -1,13 +1,16 @@
 package net.sf.openrocket.util;
 
+import java.util.ArrayList;
+import java.io.FileOutputStream;
```

Comment:
nit: this will add a warning for an unused import since the code below is commented out for reference.

Reply:
On the FileOutputStream -- nice catch.  Fixed
On the java.util.ArrayList -- the code using that isn't commented out.